src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -96,21 +96,23 @@ public function cgetAction(Request $request)
             $offset = ($request->get('page', 1) - 1) * $limit;
             $ids = $request->get('ids');
             $search = $request->get('search');
-            if ($ids !== null) {
-                $ids = explode(',', $ids);
-            }
             $types = $request->get('types');
             if ($types !== null) {
```

Comment:
We have decided on a format of `/some/url?ids=3,5,6,7`, with your suggestions we would have to change that to `/some/url?id=3&id=5&id=6&id=7`, which is not as comfortable to type